lib/ood_core/batch_connect/template.rb

```diff
@@ -95,7 +101,7 @@ def bash_helpers
           context.fetch(:bash_helpers) do
             min_port    = context.fetch(:min_port, 2000).to_i
             max_port    = context.fetch(:max_port, 65535).to_i
```

Comment:
You pulled out `passwd_size` into its own method, which is nice. Would it improve code readability to do the same with min_port and max_port?